app/policies/rubygem_policy.rb

```diff
@@ -34,7 +34,7 @@ def show_events?
   def request_ownership?
     return false if rubygem_owned_by?(user)
     return true if rubygem.ownership_calls.any?
-    return deny("above maximum downloads to be considered abandoned") if rubygem.downloads >= ABANDONED_DOWNLOADS_MAX
+    return false if rubygem.downloads >= ABANDONED_DOWNLOADS_MAX
```

Comment:
This was just an idea I was trying. I meant to remove it from all commits but I must have rebased incorrectly.

The result is the same, but we could revisit this if it's interesting to anyone.